randrctl/xrandr.py

```diff
@@ -35,7 +35,8 @@ class Xrandr:
     MODE_REGEX = re.compile("(\d+x\d+)\+(\d+\+\d+)")
     CURRENT_MODE_REGEX = re.compile("\s*(\S+)\s+([0-9\.]+)(.*$)")
 
-    def __init__(self, display: str, xauthority: str):
+    def __init__(self, display: Optional[str], xauthority: Optional[str], executable: str = "/usr/bin/xrandr"):
```

Comment:
WDYT about alternatively relying on `xrandr` being in `PATH`?

If I were writing this today, I'd definitely go with reliance on `PATH`

Reply:
I think actually that would solve most of my problem. I'm using it with just `"xrandr"` so that'd be fine.